### ISSUE TITLE
Zustände können nicht hinzugefügt werden.

### ISSUE THREAD

**Issue by CKMartens** [ioBroker/ioBroker.scenes#28]
Platform: linux
Node.js: v8.12.0
NPM: 6.4.1
iobroker.scenes: 1.1.0

Ich kann Scenen anlegen, wenn ich aber Zustände zu den Szenen hinzufügen (mit + Symbol) will, wird das Auswahlfenster der Datenpunkte geöffnet. Wenn ich einen Datenpunkt auswähle und Hinzufügen wähle wird er allerdings nicht übernommen.

Ich habe iobroker.scenes ein paar mal über die Adminoberfläche, als auch direkt in der Shell de- und instaliert. Im Logfile kam leider keine Fehlermeldung.

**Comment by BuZZy1337** [comment]
Nur um sicher zu gehen:
Mit auswählen meinst du, dass du den Haken vorne vor dem Namen vom Datenpunkt gesetzt hast?

**Comment by CKMartens** [comment]
Ja, der Hacken wird gesetzt, der Datenpunkt wird dann aber nicht übernommen.
